src/pages/version/NameChangeModal/getFormSchema.ts

```diff
@@ -0,0 +1,28 @@
+import { SpruceFormProps } from "components/SpruceForm/types";
+import { LeafyGreenTextArea } from "components/SpruceForm/Widgets/LeafyGreenWidgets";
+
+export const getFormSchema = (
+  name: string
+): {
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  schema: {
+    type: "object" as "object",
+    properties: {
+      newPatchName: {
+        title: "New patch name",
+        type: "string" as "string",
+        default: name,
+        maxLength: 300,
+        minLength: 1,
+      },
+    },
+  },
+  uiSchema: {
+    newPatchName: {
+      "ui:widget": LeafyGreenTextArea,
```

Comment:
[nit] We register the LG widget as the default so it's not necessary to import the component itself https://github.com/evergreen-ci/spruce/blob/5b22e275c3032f086b22d3b0669d5b5531cec5fa/src/components/SpruceForm/Widgets/index.tsx#L15
```suggestion
      "ui:widget": "textarea",
```